docs/content/direct/release-notes.md

```diff
@@ -4,6 +4,9 @@ The following sections list the known issues for each release. The issue list is
 
 ## 0.21.0 and its release candidates
 
+### Patch release  0.21.1 
+This patch release mainly updates the documentation exposed under kubestellar.io . In all other aspects it is identical to 0.21.0
```

Comment:
> In all other aspects it is identical to 0.21.0

No, this is not true. There have been a few bug fixes.

I do not think that it is necessary to list them all here; GitHub automatically collects a changelog (see https://github.com/kubestellar/kubestellar/releases/tag/v0.21.0 for an example).  I might simply remove the inaccurate sentence.